tox.ini

```diff
@@ -1,26 +1,16 @@
 [tox]
 envlist =
-    py{26,27}-django{14,15,16},
-    py{32,33}-django{15,16},
-    py{27}-django{17,18},
-    py{32,33,34}-django{17,18},
+    py33-django18
+    py{27,34,35}-django{18,19}
 
 [testenv]
 commands =
     coverage run setup.py test
-    coveralls
-basepython =
-    py26: python2.6
-    py27: python2.7
-    py32: python3.2
-    py33: python3.3
-    py34: python3.4
+    sh -c 'coveralls | true'
 deps =
     coverage==3.7.1
     coveralls
-    django14: Django>=1.4,<1.5
-    django15: Django>=1.5,<1.6
-    django16: Django>=1.6,<1.7
-    django17: Django>=1.7,<1.8
```

Comment:
App initialization works differently, so the updated tests fail with django1.7, regrettably already. Since dj17 is unsupported already, and dj18 is an LTS, I don't think it'll hurt in the long run.